### PR TITLE
[Manifest ID article] Fix future-looking language and add constraints

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Fix future-looking language and add constraints

### CODE REVIEW

File: site/en/blog/pwa-manifest-id/index.md [hunk 1]
```diff
@@ -38,13 +39,11 @@ that matches the already installed PWA, it will treat that as the installed PWA.
 
 ## Browser support
 
-Support for the `id` property is expected to land in desktop Chromium-based
-browsers starting with version 96. Support for mobile (which currently uses
-the manifest url as the unique id) should land in the first half of 2022.
+Support for the `id` property landed in Chrome 96.
 
-## What should I do today?
+## What should I do if I have an app without id?
```
Comment (rachelandrew) [inline site/en/blog/pwa-manifest-id/index.md:44]:
```suggestion
## What should I do if I have an app without an `id`?
```

**Review by rachelandrew — COMMENTED** [review]
This has future language because it's a blog post. Should it be an article or is it a blog post?

**Comment by tomayac** [comment]
> This has future language because it's a blog post. Should it be an article or is it a blog post?

This should have been an article, but was a blog post at the time.

**Comment by rachelandrew** [comment]
I'd suggest moving it to articles then.

**Comment by tomayac** [comment]
> Move the article and add redirect

Done in https://github.com/GoogleChrome/developer.chrome.com/pull/7524/commits/fd6d10ae837f4cbfec72fd5662b5cb1ecf99f17b.

File: site/en/articles/pwa-manifest-id/index.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ title: Uniquely identifying PWAs with the web app manifest id property
 authors:
   - petelepage
 date: 2021-09-16
+updated: 2023-10-12
 description: |
   There's a new optional `id` property as part of the web app manifest spec,
```
Comment (rachelandrew) [inline site/en/articles/pwa-manifest-id/index.md:9]:
```suggestion
  The optional `id` property is part of the web app manifest spec,
```

File: site/en/articles/pwa-manifest-id/index.md [hunk 3]
```diff
@@ -4,6 +4,7 @@ title: Uniquely identifying PWAs with the web app manifest id property
...
   that allows you to explicitly define the identifier used for your PWA.
```
Comment (rachelandrew) [inline site/en/articles/pwa-manifest-id/index.md:10]:
```suggestion
  and allows you to explicitly define the identifier used for your PWA.
```

**Review by rachelandrew — APPROVED** [review]
I just tweaked the description as it's not very new now!

**Pull request merged**